doc/installing.rst

```diff
@@ -1,222 +1,316 @@
+development testing.
+
+Atmospheric Model Grids
+-----------------------
+
+Calculation of the sensitivity functions for IR instruments and general fitting
+of telluric absorption uses a grid of model atmosphere spectra.  These model
+grids are provided in the ``Telluric`` directory in the `PypeIt dev-suite Google
+Drive`_ and range in size from 3.5-7.7 GB.  Each file provides model spectra for
+atmospheric conditions specific to an observatory; however, a model grid is not
+provided for all observatories with spectrographs supported by ``PypeIt``.  If
+you do not find models for your observatory, you can use the Maunakea model as
+an approximation. It includes a large grid of different parameters and should be
+good enough for most purposes.
+
+To install the model grids:
+
+    #. Download the relevant file(s) from the Telluric directory in the `PypeIt
+       dev-suite Google Drive`_.
+
+    #. Run the ``pypeit_install_telluric`` script.  For example, if you've
+       downloaded ``TelFit_MaunaKea_3100_26100_R200000.fits`` to ``my_path``,
+       then you would execute:
+
+        .. code-block:: console
+
+            pypeit_install_telluric --path my_path
+
+       or
+
+        .. code-block:: console
+
+            cd my_path
+            pypeit_install_telluric
+
+The ``pypeit_install_telluric`` script simply searches for all the ``TelFit*``
+files in the path it is provided and creates symlinks to those files in the
+``pypeit/data/telluric/atm_grids`` directory in your installation of ``PypeIt``.
+
+.. warning::
 
- - Installation of ``numba`` has been known to fault because of an issue with
-   the LLVM compiler. For one particular case, the solution was to revert to
-   llvm version 9.0.1 using `Homebrew <https://brew.sh/>`_ and then add
-   environmental variables to your shell rc that point to the reverted
-   directory structure. ``numba`` also does not yet officially support
-   Python 3.9.
+    The installation script simply creates symlinks to the downloaded data.
+    This means that if you move the original data, the symlinks will become
+    broken **and you will need to rerun the installation script.**
 
- - At the moment, an implicit dependency on QT bindings remains (either PyQT5 or
-   PySide2) because of our dependence on ``linetools``.
 
- - Note that ``shapely`` is provided as an optional dependency, but is only
-   currently used by one method that calculates the spaxel area for KCWI
-   output datacubes.
+Quick-look Master Files
+-----------------------
+
+Some of the quick-look reductions provided by ``PypeIt`` require canned master
+files to speed up the data-reduction process, as appropriate for a quick-look
+result.  These files are hosted in the ``QL_MASTERS`` directory in the `PypeIt
+dev-suite Google Drive`_.
+
+To install the quick-look master files:
+
+    #. Right-click on the ``QL_MASTERS`` folder in the `PypeIt dev-suite
+       Google Drive`_ and select the "Download" option from the drop-down menu.
```

Comment:
Just a note: once this goes live the old mosfire_masters.tar.gz should be removed from the devsuite google drive, so that people aren't downloading an additonal copy with out of date information.

Reply:
Agreed!